dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/citation/CitationTestUtils.java

```diff
@@ -57,6 +58,16 @@ public DatasetVersion createATestDatasetVersion(String withTitle, boolean withAu
         return datasetVersion;
     }
 
+    public DatasetVersion createHarvestedTestDatasetVersion(String withTitle, boolean withAuthor) {
+        try {
+            DatasetVersion datasetVersion = createATestDatasetVersion(withTitle, withAuthor);
+            datasetVersion.getDataset().setHarvestedFrom(new HarvestingClient());
+            return datasetVersion;
+        } catch (ParseException pe) {
+            return null;
```

Comment:
I would avoid catching exceptions in code for tests. Simply adding throws in method declaration will cause that when something is wrong in the code it would be easier to detect the cause of it